source/release-notes/backend.txt

```diff
@@ -13,6 +13,22 @@ Atlas App Services Changelog
 2023 Releases
 -------------
 
+.. _backend_20230524:
+
+24 May 2023 Release
+~~~~~~~~~~~~~~~~~~~
+
+- Increased function :ref:`execution timeout <function-constraints>`
+  from 270 seconds to 300 seconds.
+
+- Added support for automatically migrating a Device Sync App from
+  partition-based sync to flexible sync.
```

Comment:
For consistency with the [migration docs](https://www.mongodb.com/docs/atlas/app-services/sync/migrate-sync-modes/), I guess we should capitalize similarly: 
```suggestion
  Partition-Based Sync to Flexible Sync.
```

Related, do you want to add a link to those docs here?